src/utils/WellKnownUtils.ts

```diff
@@ -63,7 +63,7 @@ export function isE2EEEnabledInWellKnown(): boolean {
 }
 
 export function isE2EEDefaultedOnInWellKnown(): boolean {
-	// If encryption is disabled then don't default encrpyion on
+	// If encryption is disabled then don't default encryption
```

Comment:
I'm not sure why this change is required, as the comment now doesn't make sense.

Reply:
Change has been reverted. I was confused little bit initially.